utils/auto_update.py

```
@@ -184,174 +187,138 @@
     parser = argparse.ArgumentParser(description="DeepSeek Desktop Auto-Updater")
-    parser.add_argument(
-        "--auto",
-        action="store_true",
-        help="Run in auto mode (non-interactive)."
-    )
+    parser.add_argument("--auto", action="store_true", help="Run in auto mode (non-interactive).")
+    parser.add_argument("--debug", action="store_true", help="Keep console open after update for debugging.")
     args = parser.parse_args()
 
     auto_mode = args.auto
+    debug_mode = args.debug
```

Ruff (0.12.2)

Local variable `debug_mode` is assigned to but never used

Remove assignment to unused variable `debug_mode`

(F841)